### PR TITLE
Added Locust and Nighthawk to the Tools Section of the Landscape and …

### LINKED ISSUE #417: [Landscape] Add Locust to the Tools section of the Service Mesh Landscape
**Current State:**
Locust is a python-based an open source, distributed load testing tool.

**Desired State:**
Locust should be included in the service mesh landscape under the [tools section](https://layer5.io/landscape#tools).

Related to https://github.com/tsenart/vegeta/issues/510

---
**Contributor Resources**
- [Instructions for updating the landscape](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#updating-the-service-mesh-landscape)

### PR DESCRIPTION
**Pull request by ashiscs**
…added Vegeta github link
fixes #417 , #415  
Signed-off-by: ashiscs <ashissingh640@gmail.com>

### CODE REVIEW

File: _includes/landscape/service-mesh-tools.html [hunk 1]
```diff
@@ -99,12 +99,28 @@
...
+            <td>Scalable user load testing tool</td>
+            <td>Locustio </td>
+            <td style="white-space: pre-wrap;">Locust is an easy-to-use, distributed, user load testing tool. It is intended for load-testing web sites (or other systems) and figuring out how many concurrent users a system can handle.
+            </td>
+          </tr>
+          <tr style="border: 1px dashed #a6b7be; text-align:center;">
+            <td><a href="https://github.com/envoyproxy/nighthawk">Nighthawk</a></td>
+            <td>C++</td>
+            <td>L7 (HTTP/HTTPS/HTTP2) performance characterization tool</td>
+            <td></td>
```
Comment (mum4k) [inline _includes/landscape/service-mesh-tools.html:121]: Can we put "general use" in here?

File: _includes/landscape/service-mesh-tools.html [hunk 2]
```diff
@@ -99,12 +99,28 @@
...
+            <td style="white-space: pre-wrap;">A L7 (HTTP/HTTPS/HTTP2) performance characterization tool.
```
Comment (mum4k) [inline _includes/landscape/service-mesh-tools.html:122]: The first sentence is good. Can we also add:

Its design focuses on exact request release timing and aims to provide its users with the ability to dynamically customize the request headers and content during an experiment.

**Comment by ashiscs** [comment]
@mum4k sir can you check the recent update !

File: _includes/landscape/service-mesh-tools.html [hunk 3]
```diff
@@ -99,12 +99,28 @@
...
+            <td>Locustio </td>
+            <td style="white-space: pre-wrap;">Locust is an easy-to-use, distributed, user load testing tool. It is intended for load-testing web sites (or other systems) and figuring out how many concurrent users a system can handle.
+            </td>
+          </tr>
+          <tr style="border: 1px dashed #a6b7be; text-align:center;">
+            <td><a href="https://github.com/envoyproxy/nighthawk">Nighthawk</a></td>
+            <td>C++</td>
+            <td>L7 (HTTP/HTTPS/HTTP2) performance characterization tool</td>
+            <td>General use</td>
+            <td style="white-space: pre-wrap;">A L7 (HTTP/HTTPS/HTTP2) performance characterization tool.Its design focuses on exact request release timing and aims to provide its users with the ability to dynamically customize the request headers and content during an experiment.
```
Comment (mum4k) [inline _includes/landscape/service-mesh-tools.html:122]: (nit) Missing a space after "characterization tool."

File: _includes/landscape/service-mesh-tools.html [hunk 4]
```diff
@@ -99,12 +99,28 @@
...
+            <td>Scalable user load testing tool</td>
+            <td>Locustio </td>
+            <td style="white-space: pre-wrap;">Locust is an easy-to-use, distributed, user load testing tool. It is intended for load-testing web sites (or other systems) and figuring out how many concurrent users a system can handle.
+            </td>
+          </tr>
+          <tr style="border: 1px dashed #a6b7be; text-align:center;">
+            <td><a href="https://github.com/envoyproxy/nighthawk">Nighthawk</a></td>
+            <td>C++</td>
+            <td>L7 (HTTP/HTTPS/HTTP2) performance characterization tool</td>
+            <td>General use</td>
```
Comment (mum4k) [inline _includes/landscape/service-mesh-tools.html:121]: (consistency nit) I think all the other tools in this list used lowercase, i.e. "general use".

**Pull request merged**